src/crimson/os/seastore/async_cleaner.cc

```diff
@@ -963,6 +963,8 @@ void SegmentCleaner::register_metrics()
 		    sm::description("the number of projected usage reservations")),
     sm::make_counter("projected_used_bytes_sum", stats.projected_used_bytes_sum,
 		    sm::description("the sum of the projected usage in bytes")),
+    sm::make_counter("projected_used_bytes", stats.projected_used_bytes,
+		    sm::description("the current projected usage in bytes")),
```

Comment:
See another comment, I think this is duplicated to the `projected_used_bytes_sum` + `projected_count` combination.